app/services/idv/gpo_mail.rb

```diff
@@ -2,14 +2,16 @@ module Idv
   class GpoMail
     MAX_MAIL_EVENTS = IdentityConfig.store.max_mail_events
     MAIL_EVENTS_WINDOW_DAYS = IdentityConfig.store.max_mail_events_window_in_days
+    MINIMUM_WAIT_BEFORE_ANOTHER_USPS_LETTER_IN_HOURS =
+      IdentityConfig.store.minimum_wait_before_another_usps_letter_in_hours
 
     def initialize(current_user)
       @current_user = current_user
     end
 
     def mail_spammed?
-      return false if user_mail_events.empty?
-      max_events? && updated_within_last_month?
+      rate_limiting_enabled? &&
+        (too_many_mails_within_window? || last_mail_too_recent?)
```

Comment:
Since the individual checks return false when that check is disabled, you don't need to check for rate_limiting here. I see that you're specifically testing for turning off both checks by changing one of the config settings to zero. Is that an explicit requirement? I think they should stay independent. It's unexpected behavior for them to be linked, and that's an awful lot of specs to confirm that it works that way.
```suggestion
        too_many_mails_within_window? || last_mail_too_recent?
```

Reply:
In a "does the code work" sense, you are of course correct. That's there for the benefit of humans; it makes the code more clearly expressive of when rate limiting is disabled. The checks for rate limiting being disabled if either the window time or the events within the window were pre-existing; I merely updated them to use GpoConfirmationCodes instead of UserEvents

Reply:
Per slack conversation: No it doesn't.